storybook/src/elements/nuxeo-document-layout/nuxeo-document-layout.stories.js

```diff
@@ -0,0 +1,174 @@
+import { storiesOf } from '@storybook/polymer';
+import { html } from 'lit-html';
+import { until } from 'lit-html/directives/until.js';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
+import { select } from '@storybook/addon-knobs';
+import { LayoutBehavior } from '@nuxeo/nuxeo-ui-elements/nuxeo-layout-behavior.js';
+import { Polymer } from '@polymer/polymer/lib/legacy/polymer-fn.js';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-document-layout.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-input.js';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-date-picker.js';
+import hljs from 'highlight.js';
+import '../../../.storybook/i18n';
+import image from '../../img/image01.jpg';
+import DocumentBuilder from '../../data/documents.data';
+/* import '../code-highlighter.js'; */
+
+window.Polymer = Polymer;
```

Comment:
This makes me think we could have some sort on compat or legacy js file in elements instead of in Web UI 🤔

Reply:
We could... 🤔